test/scenarios/17. settings/advanced/show-test-networks.csv

```diff
@@ -0,0 +1,16 @@
+Advanced settings: Check “Show test networks” toggle works correctly,,,,
+Step,Test Steps,Test Data,Expected Result,Notes
+1,Open the extension.,,The Welcome Back screen is shown,
+2,Proceed to Unlock the wallet.,password (8 characters min),,
+3,Click on the right corner setting(three dots),,Drop down menu should appear,
+4,Click on the Settings from menu.,,Settings page appears,
+5,Click on the Advanced tab.,,Advanced settings page appears,
+6,Validate that the 'Show test networks' is default to off.,,The toggle is set to off by default.,
+7,Click on the Network selection drop down on the left top,,Select a network dialog box appears,
+8,Validate that the 'Show test networks' toggle,,The 'Show test networks' toggle is off,
```

Comment:
Could you add validation that when toggle is off, test networks are not shown in the drop down list?